index.html

```diff
@@ -27,125 +27,144 @@
         id="layoutApp">
 
         <div
-          class="layoutJSON">
+          class="navigationArea">
           <div class="row">
-            <div class="col-xs-9">
-              <input type="checkbox" v-model="kioskMode"/> kioskMode
-              <input type="checkbox" v-model="authorMode"/> authorMode
+            <div class="col-xs-10">
               <button
                 v-for="t in tableaux"
                 class="btn btn-default btn-xs tableau-button"
                 v-bind:class="{ current: isCurrentTableau(t) }"
-                v-on:click="loadTableau(t)">
+                v-bind:disabled="isCurrentTableau(t)"
```

Comment:
What's the `disabled` attribute here for? I can't find its definition in the css file.

Reply:
`disabled` is not a CSS thing, it's an HTML thing. In this case, we want to disable the button if we are already on that tableau. Another example would be where we want to have a `Send` button disabled unless an associated input field is non-empty. See https://www.w3schools.com/tags/att_button_disabled.asp